packages/fxa-payments-server/src/styles/tailwind.css

```diff
@@ -26,7 +26,7 @@
 }
 
 .payment-panel {
-  @apply mx-4 tablet:mt-6 tablet:max-w-xs tablet:col-span-2 tablet:row-start-1 tablet:row-end-4;
+  @apply mb-6 tablet:mt-6 tablet:min-w-[18rem] tablet:max-w-xs tablet:col-start-2 tablet:col-end-auto tablet:row-start-1 tablet:row-end-3;
```

Comment:
Same as above, should this be `tablet:min-w-72`?

Reply:
Same as above - I'll look to see if it appears in DevTools when written that way.